source/server-discovery-and-monitoring/tests/errors/error_handling_handshake.yml

```diff
@@ -5,6 +5,7 @@ phases:
   responses:
   - - a:27017
     - ok: 1
+      helloOk: true
```

Comment:
That makes sense. Since `helloOk` is always sent in monitoring, new servers will always return this in monitoring responses.

Can you also update the `*.template` files in this directory and regenerate with generate-error-tests.py to ensure those are in sync?

Reply:
Good catch. I updated the `yml` files and then regenerated the `json` files, but forgot about the `template` files.

Reply:
Ran `generate-error-tests.py` and confirmed that everything is in sync.